### DISCUSSION TITLE
Panorama

### DISCUSSION THREAD

**Discussion by cycyx** [deckerst/aves#1125] [Q&A] [answered]
I'm wondering how Aves identifies panorama pictures.

When using "dedicated" panorama app, like Panorama360, thr file is detected as a panorama, is tagged and can be displayed in the dedicated viewing mode.

If I make a panorama using the "pano" mode of OpenCamera though, the file is not identified/tagged as a panorama.

This may be an issue from OpenCamera, but I would like to know more about your detection/analyze method.

**Comment by deckerst** [comment, marked as answer]
Indeed, a panorama is not simply a long landscape image. It relies on metadata. 

If you're curious about the actual code, see [this](https://github.com/deckerst/aves/blob/cfe4c72e8f9fa1f07ea20a00a3b4f7633a9fe831/android/app/src/main/kotlin/deckers/thibault/aves/metadata/xmp/XMP.kt#L158).

You can check your own images that are detected as panoramas. In the Info page, check the XMP section and you'll see panorama related metadata.

**Comment by cycyx** [comment]
Ok, thank you for your answer(s).

Now to get OpenCamera developers to create the requited metadata! ^^
